web/src/components/storage/ProposalSettingsSection.jsx

```diff
@@ -112,6 +112,65 @@ const EncryptionSettingsForm = ({
   );
 };
 
+/**
+ * Allows to define snapshots enablement
+ * @component
+ *
+ * @param {object} props
+ * @param {ProposalSettings} props.settings - Settings used for calculating a proposal.
+ * @param {onChangeFn} [props.onChange=noop] - On change callback
+ *
+ * @callback onChangeFn
+ * @param {object} settings
+ */
+const SnapshotsField = ({
+  settings,
+  onChange = noop
+}) => {
+  const rootVolume = (settings.volumes || []).find((i) => i.mountPath === "/");
+
+  const initialChecked = rootVolume !== undefined && rootVolume.fsType === "Btrfs" && rootVolume.snapshots;
```

Comment:
I would feel safer if that comparison of `fsType` would be case insensitive.

Reply:
> I would feel safer if that comparison of fsType would be case insensitive.

Then, please move it to a `isBtrfs` util method.

Reply:
>  please move it to a isBtrfs util method

https://github.com/openSUSE/agama/blob/master/web/src/components/storage/utils.js looks as a good location for placing it